src/common/pascalscript/Source/uPSRuntime.pas

```diff
@@ -4227,7 +4227,7 @@ procedure PSDynArraySetLength(var arr: Pointer; aType: TPSTypeRec; NewLength: Lo
     end;
     GetMem(darr, Longint(NewLength * elSize) + SizeOf(TDynArrayRecHeader));
     {$IFDEF CPUX64}
-    darr^.header._Padding:=0;
+    //0xdarr^.header._Padding:=0; //We ignore x64 machines

```

Comment:
Why did it enter there if it is surrounded with CPUX64 ? 

Please read this messages last paragraph: http://www.knightsandmerchants.net/forum/viewtopic.php?p=38197&sid=b62bf461852f14aa3f1056e593eedb88#p38197

It looks like you have x64 Lazarus with x64 target. KMR should be built with x32 for x32